gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -40,6 +40,20 @@
 #define ACCL_NPU_SRCN_STR  "npu.srcn" /** srcn hardware supported by nnfw */
 #define ACCL_NPU_SR_STR  "npu.sr"
 
+#define GST_TENSOR_FILTER_FRAMEWORK_V0 (0xDEAFDEAD00000000ULL)
+#define GST_TENSOR_FILTER_FRAMEWORK_V1 (0xDEAFDEAD00010000ULL)
+
+/** TODO: update this to 1 after supporting version 1 GstTensorFilterFramework in tensor_filter.c */
+#define GST_TENSOR_FILTER_API_VERSION_DEFINED (0)
+#define GST_TENSOR_FILTER_API_VERSION_MIN (0)	/* The minimum API version supported (could be obsolete) */
+#define GST_TENSOR_FILTER_API_VERSION_MAX (0)	/* The maximum API version supported (recommended) */
+
+/**
+ * @brief Check the value of the version field of GstTensorFilterFramework
+ */
+#define checkGstTensorFitlerFrameworkVersion(value, version) \
+  ((0xDEAFDEAD00000000ULL | ((version) << 16)) == (value & 0xFFFFFFFFFFFF0000ULL))
```

Comment:
How about defining `0xDEAFDEAD00000000UL` for readability? (and can be used in both `GST_TENSOR_FILTER_FRAMEWORK_V0/1`)

Reply:
Appears good.

Reply:
Updated.